src/encoded/tests/test_types_file_set.py

```diff
@@ -123,3 +127,12 @@ def test_validate_compatible_assay_and_sequencer_on_post(
     }
     post_item(es_testapp,post_body,'file_set',status=expected_status)
 
+
+@pytest.mark.workbook
+def test_associated_files_retracted(es_testapp: TestApp, workbook: None) -> None:
+    """Ensure `associated_files_retracted` calc_prop is working."""
+    status_search = get_search(
+        es_testapp,
+        "?type=FileSet&files_status_retracted=True",
+    )
+    assert status_search
```

Comment:
I would do a count check here at least to ensure you're getting the right search results